server/middlewares/articles/findArticle.js

```diff
@@ -0,0 +1,33 @@
+import models from "../../models";
+import { serverError, httpResponse } from "../../helpers/http";
+
+const { Articles } = models;
+/**
+ * @description Middleware that search an Article by slug and 
+              returns the article, error code 404 if not found
+ * @param  {httpRequest} req
```

Comment:
What is httpRequest? A string,  an array or boolean? Replace it with object and add text to describe req after it. Do the same thing in all similar cases